scripts/helpers/json.js

```diff
@@ -0,0 +1,3 @@
+'use strict'
```

Comment:
Same, I think this can be removed.

Even if we want to keep these for future use, I think it'd be better to create something like `helpers/common.js`
And export all these methods from one file instead of creating a separate file for each method.

Reply:
Doesn't work like that with metalsmith-discover-helpers/partials, which are in turn dependencies for metalsmith-layout.